apps/web/src/components/transactions/TxFilterForm/index.tsx

```diff
@@ -189,7 +191,9 @@ const TxFilterForm = ({ toggleFilter }: { toggleFilter: () => void }): ReactElem
                               className={inputCss.input}
                               label={
                                 fieldState.error?.message ||
-                                `Amount${isMultisigFilter && chain ? ` (only ${chain.nativeCurrency.symbol})` : ''}`
+                                (isIncomingFilter
+                                  ? 'Amount (with decimals)'
```

Comment:
Not sure if "with decimals" is clear enough. Maybe it should say "with zeroes"?

Reply:
I think "with zeros" is clearer because "decimals" suggest a float. We also could add a tooltip explaining it's the "base amount".

Reply:
Done:
<img width="825" alt="Screenshot 2025-02-11 at 15 25 33" src="https://github.com/user-attachments/assets/2ae4b36c-9ff5-43ee-b1fc-09d269ba8a36" />